docs/topics/Terms-of-Use.md

```diff
@@ -0,0 +1,23 @@
+# Terms of Use
+
+By using the o!TR API, you agree to abide by these terms of use. The o!TR API is provided publicly, for free, for all users by the osu! Tournament Rating Team ("o!TR Team", "us", "our").
+
+## Scraping
+
+The API must not be used to scrape information. Scraping data, such as tournaments, matches, etc. is forbidden and will result in a permanent revocation of API access. Please see the [data](Data.md) page for more information on conveniently importing our dataset.
+
+## Accreditation
+
+Data which is sourced from us should be credited properly. Misleading users to believe data created by us is provided by another source, or failing to credit this information as being created by us, is not allowed.
```

Comment:
do *we* mention (when relevant) how we source our data?

Reply:
Could probably go under the Data page? Somewhere under the primary section at least.

Reply:
For now I just added one extra sentence about this. Page can come later.